MediaWikiKit/MediaWikiKit/MWKDataStore.h

```diff
@@ -154,4 +154,6 @@ extern NSString* const MWKArticleKey;
 
 - (NSError*)removeFolderAtBasePath;
 
+- (BOOL)htmlFileExistsForSectionId:(NSUInteger)sectionId title:(MWKTitle*)title;
```

Comment:
Cocoa comment: the convention for methods returning `BOOL` to have a verb prefix (e.g. `is`, `has`, `contains`). For example: `-[UIViewController isViewLoaded]`, `-[NSString hasPrefix:]`, and `-[NSArray containsObject:]`.